docs/resources/remote_jetbrainsplugins_repository.md

```
@@ -0,0 +1,62 @@
+
+The following attributes are **not** part of the JetBrains Plugins contract and are not exposed —
+Artifactory ignores or rejects them for this package type:
+
+* `external_dependencies_enabled` / `external_dependencies_patterns` (not annotated for this type)
+* `enable_token_authentication` (annotated for Docker / OCI / Helm OCI only)
+* `curated` / `pass_through` (Curation is not supported for JetBrains Plugins; setting `curated = true` returns HTTP 400)
+
+## Import
+
+Remote repositories can be imported using their name, e.g.
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)